mass_mailing_newsletter_welcome_mail/data/base_automation_data.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <data>
+        <record id="email_template" model="mail.template">
+            <field name="name">Welcome mail to newsletter subscribers</field>
+            <field name="model_id" ref="mass_mailing.model_mail_mass_mailing_contact"/>
+            <field name="use_default_to" eval="True"/>
+            <field name="lang">${object.env.context.get("lang")}</field>
+            <field name="subject">Welcome to the mailing list</field>
```

Comment:
```suggestion
            <field name="subject">Welcome to the mailing list ${object.list_id.name}</field>
```

Reply:
We changed this yesterday, customizing only the Newsletter list. That's the reason we re-named the module. I understand that this is a good contribution for any list, even if it's only for Newsletter, it's the same.